todo-frontend-angular/src/app/todo/todolist/todolist.component.ts

```diff
@@ -23,12 +25,30 @@ export class TodolistComponent implements OnInit {
       marginTop: '30px'
     }
   };
-
+    addBtnConfig: ButtonConfiguration= {
+      styles: {
+        position: 'relative',
+        width: '150px',
+        height: '60px',
+        backgroundColor: '#f92672',
+        color: '#fff',
+        fontFamily: 'sans-serif',
+        fontSize: '20px',
+        borderRadius: '10px',
+        marginTop: '30px',
+        
+      }
+    };
   constructor(private readonly todoService: TodoService) { }
 
   ngOnInit(): void {
   }
   onClickEventReceived() {
```

Comment:
Look to task number 6 :) See if you can spot what should be done with your @outputs and eventhandlers

Reply:
I think Nicolai did this on another branch